CONTRIBUTING.md

```diff
@@ -350,7 +350,23 @@ by running the test suite in a Docker environment that simulates Swift on Linux.
     cd swift-docc
     swift run docc
     ```
-    
+
+## Updating Build Rules
+
+In order to build DocC as part of the toolchain distribution, a parallel CMake
```

Comment:
Is there some information missing in this first sentence? DocC has been part of the Swift toolchain on both Linux and macOS for several years without this CMake build, so it feels incorrect, or at least confusing, to me to say that the CMake build is required for DocC to be part of the toolchain distribution.

Reply:
I guess we can add explicitly for Windows if you like - if you are bootstrapping the toolchain, you cannot build with SPM on all platforms.